### DISCUSSION TITLE
"Internal Error" showing when trying to add a SSL Certificate in Proxy Host.

### DISCUSSION THREAD

**Discussion by iahadaman** [NginxProxyManager/nginx-proxy-manager#1555] [Q&A]
Hello everyone! I hope this message finds you in good health.

First of all, this is not my first time using proxy manager but IT IS the first time I'm facing this issue.
So I installed web radio (AzuraCast) on port 90 and then install the Nginx proxy manager on port 80.

I then, from my web hosting panel pointed an A record to the VPS IP.
It was working and the subdomain was showing the default Nginx page.

The problem occurs when I try to add the SSL.
I log in to NGINX Admin panel > go to proxy hosts > add New > fill the domain and IP and port (90) >and then go to the SSL tab.

When I select the options on this tab and click save it's giving me an "internal error" that's it.

Please, let me know if there is a solution to this problem. Thank you!

**Comment by chaptergy** [comment]
Could you please check the logs of the docker container so see if it contains any useful information?

**Comment by iahadaman** [comment]
Can you please tell me where I can find the logs?

**Reply by chaptergy** [reply]
See [docker docs](https://docs.docker.com/engine/reference/commandline/logs/) for the logs of the container and possibly https://github.com/jc21/nginx-proxy-manager/issues/1271#user-content-certificate-error to get the letsencrypt logs if they contain anything useful.
